cmd/state-svc/internal/rtwatcher/watcher.go

```diff
@@ -97,7 +97,7 @@ func (w *Watcher) check() {
 
 func (w *Watcher) RecordUsage(e entry) {
 	logging.Debug("Recording usage of %s (%d)", e.Exec, e.PID)
-	w.an.Event(anaConst.CatRuntimeUsage, anaConst.ActRuntimeHeartbeat, e.Dims)
+	w.an.EventWithSource(anaConst.CatRuntimeUsage, anaConst.ActRuntimeHeartbeat, anaConst.SrcExecutor, e.Dims)
```

Comment:
It looks to me like all heartbeats come from executors, so that's why the source is this and not State Tool.

Reply:
Good call! I forgot about that caveat.